From 63187bca9ce8888090fc4a73135f56bd927b9ee0 Mon Sep 17 00:00:00 2001
From: Vitaly Andrianov <vitalya@ti.com>
Date: Mon, 28 Aug 2017 17:17:38 -0400
Subject: [PATCH 54/86] arm: dtb: am572x-idk: add dts files to support
 jailhouse inmates

This commit adds dts files to support jailhouse inmate on am572x-idk
with 2 supported LCDs and w/o LCD.

Signed-off-by: Vitaly Andrianov <vitalya@ti.com>
---
 arch/arm/boot/dts/am572x-idk-jailhouse.dts          | 10 ++++++++++
 arch/arm/boot/dts/am572x-idk-lcd-osd101t2045-jh.dts | 10 ++++++++++
 arch/arm/boot/dts/am572x-idk-lcd-osd101t2587-jh.dts | 10 ++++++++++
 3 files changed, 30 insertions(+)
 create mode 100644 arch/arm/boot/dts/am572x-idk-jailhouse.dts
 create mode 100644 arch/arm/boot/dts/am572x-idk-lcd-osd101t2045-jh.dts
 create mode 100644 arch/arm/boot/dts/am572x-idk-lcd-osd101t2587-jh.dts

diff --git a/arch/arm/boot/dts/am572x-idk-jailhouse.dts b/arch/arm/boot/dts/am572x-idk-jailhouse.dts
new file mode 100644
index 0000000..733f7ce
--- /dev/null
+++ b/arch/arm/boot/dts/am572x-idk-jailhouse.dts
@@ -0,0 +1,10 @@
+/*
+ * Copyright (C) 2017 Texas Instruments Incorporated - http://www.ti.com/
+ *
+ * This program is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License version 2 as
+ * published by the Free Software Foundation.
+ */
+
+#include "am572x-idk.dts"
+#include "am572x-idk-jailhouse.dtsi"
diff --git a/arch/arm/boot/dts/am572x-idk-lcd-osd101t2045-jh.dts b/arch/arm/boot/dts/am572x-idk-lcd-osd101t2045-jh.dts
new file mode 100644
index 0000000..4b21fc8
--- /dev/null
+++ b/arch/arm/boot/dts/am572x-idk-lcd-osd101t2045-jh.dts
@@ -0,0 +1,10 @@
+/*
+ * Copyright (C) 2017 Texas Instruments Incorporated - http://www.ti.com/
+ *
+ * This program is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License version 2 as
+ * published by the Free Software Foundation.
+ */
+
+#include "am572x-idk-lcd-osd101t2045.dts"
+#include "am572x-idk-jailhouse.dtsi"
diff --git a/arch/arm/boot/dts/am572x-idk-lcd-osd101t2587-jh.dts b/arch/arm/boot/dts/am572x-idk-lcd-osd101t2587-jh.dts
new file mode 100644
index 0000000..cab0c92
--- /dev/null
+++ b/arch/arm/boot/dts/am572x-idk-lcd-osd101t2587-jh.dts
@@ -0,0 +1,10 @@
+/*
+ * Copyright (C) 2017 Texas Instruments Incorporated - http://www.ti.com/
+ *
+ * This program is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License version 2 as
+ * published by the Free Software Foundation.
+ */
+
+#include "am572x-idk-lcd-osd101t2587.dts"
+#include "am572x-idk-jailhouse.dtsi"
-- 
1.9.1

